forge/csrc/passes/pre_lowering_passes.cpp

```diff
@@ -366,7 +363,7 @@ void fuse_bias(Graph *graph)
                 broadcast = true;
                 break;
             }
-
```

Comment:
Make sure to run pre-commit checks as well :)) 

For more details, check this doc segment: https://docs.tenstorrent.com/tt-forge-fe/tools.html